test/remittance.js

```diff
@@ -7,23 +7,23 @@ const zeroAddress = '0x0000000000000000000000000000000000000000';
 
 contract('Remittance', accounts => {
   const [owner, carol, someoneElse] = accounts;
-  let contract;
+  let remittance;
 
   const bobCorrectPassword = fromAscii('bob-super-secure-password');
   const carolCorrectPassword = fromAscii('carol-super-secure-password');
 
   const bobWrongPassword = fromAscii('bob-wrong-password');
   const carolWrongPassword = fromAscii('carol-wrong-password');
 
-  beforeEach(async () => {
-    contract = (await Remittance.new(false, { from: owner, gas: 3000000 })).contract;
+  beforeEach('creating new instance', async () => {
+    remittance = (await Remittance.new(false, { from: owner, gas: 3000000 })).contract;
   });
 
   context('constructor()', () => {
-    it('should deploy the contract correctly', async () => {
-      assert.ok(contract);
-      assert.strictEqual(await contract.methods.getOwner().call(), owner);
-      assert.strictEqual(await contract.methods.isPaused().call(), false);
+    it('should deploy the remittance correctly', async () => {
+      assert.ok(remittance);
```

Comment:
I think that's a poor test. Normally `remittance.transactionHash` could yield better info.